### ISSUE TITLE
Profile option dont redirect -- only text redirect

### ISSUE THREAD

**Issue by jsaumap** [systeclabs/sinapsys#11]
in user menu "my profile" when click button dont redirect to profile, but if click text "my profile" redirect

![Captura](https://user-images.githubusercontent.com/16040298/136428819-1bc5e5bc-71e7-447f-b17a-5355117d307b.PNG)

**Comment by bkerz** [comment]
Yes, that's because the both the icon and the text are wrapped by a div, so to fix this the only thing we need to do is to reorder the elements and put the icon inside the link element `<a></a>`.

Another possible solution is to add `position:absolute;` to the icon and some styles to add more width to the link for it to be in front of the icon. In this way it won't matter if you click to the icon you will click to the link as well because it would be in front or up to the icon.
![image](https://user-images.githubusercontent.com/48168047/136463235-b3c46665-5000-47df-b577-bed956be2cb9.png)
